jld-day15-parser/Cargo.toml

```diff
@@ -2,5 +2,6 @@
 name = "day15"
 version = "0.1.0"
 authors = ["Jed Davis <jld@panix.com>"]
+
 [dependencies]
-combine = "1.2.0"
+combine = "= 1.3.0"
```

Comment:
It's intentional. In every case I looked to see what Cargo was already using (by looking at the `Cargo.lock` file) and just used that version number. Mostly it was the patch version number that changed (or was added), but in one or two cases the minor version number changed too.